CamGuesser/app/Http/Controllers/APIController.php

```diff
@@ -20,51 +21,43 @@ private function httpWithHeaders(): PendingRequest
         ]);
     }
 
-    private function getRequestToApi($requestParameters = null): string
+    private function urlWithParameters($requestParameters = null): string
```

Comment:
This class is not a controller but a service. To be more specific, it is a `WindyClient`.